### DISCUSSION TITLE
Roadmap for Zen

### DISCUSSION THREAD

**Discussion by spectrum705** [zen-browser/desktop#1882] [Polls]
can we have a roadmap page on site maybe?  like, whats being worked on , what's being considered soon, and maybe future goals? I know you do really frequent status update but that would be nice to have as well? 
 we can avoide minor features or bug fixes. But anything that's of more convenient/anticipated to user, include major things being worked on example tab groups? 
. Like we could have it this way, what's being worked on for next update(short term), the what's being considerate for next months,(things that take a little time ) etc , then the ones that are planned for future (long term )

we can use any services like notion etc, or put it on the main site.
what do you guys think? please share your views and suggestions. thank you :)

**Comment by RealDishwash** [comment]
I think it would be easier to track issues/discussions if we had a roadmap on github projects. +1 from me

**Comment by mikedlr** [comment]
Not sure how that fits into an open source project where people work on what they are interested in when they feel they have time. Definitely it has to add to people's happiness and satisfaction without putting them under pressure. It might be that someone volunteering to make something clever for them would help?

**Comment by jeanlucthumm** [comment]
This sounds good from the user side but annoying to maintain from the dev side

**Comment by spitefully-positive** [comment]
A roadmap would be awsome, but I could go without one if it creates to much friction during development.
(Coming from a non contributing consumer)

**Comment by gomster** [comment]
like this ? - https://github.com/orgs/zen-browser/projects/4

**Reply by BraisonCrece** [reply]
This is a project kanban panel, not a roadmap, isn't it?

**Comment by j13ag0** [comment]
Having a roadmap means that the (thought) leaders who created it are looking forward, not just building what feels like a fun thing to do that day. Software developers are not all motivated by daily whims. Many great devs are driven by building the things that significantly move a project to an envisioned place.
A roadmap provides a shared view of where the project is going and helps to set development priorities.
With the understanding that open source projects differ from commercial ones, its reasonable to say that both benefit from a sense of direction. 
A roadmap doesn't have to be written in stone. Its even more useful if its get reviewed on some not too frequent regular basis.
